uaber-api/uaber/services/driver.py

```diff
@@ -14,3 +14,23 @@ async def find_drivers() -> List[Driver]:
 async def create_driver(driver: Driver) -> Driver:
     await db.drivers.insert_one(driver.dict())
     return driver
+
+
+async def check_in_driver(driver_id: str):
+    driver_check_in = DriverCheckIn() # todo: add localization
+    update_result = await db.drivers.update_one({"_id": driver_id}, {"check_in": driver_check_in})
```

Comment:
Driver has his `uid` already. I think, at this point it will be better to use an `UUID` instead of `ObjectId`. With `ObjectID` we are going to isolate this API only for MongoDB.